src/linux/classes/com/alibaba/wisp/engine/WispTask.java

```diff
@@ -245,14 +245,14 @@ protected void run() {
                     } finally {
                         assert timeOut == null;
                         assert controlGroup == null; // detached
-                        runnable = null;
                         WispEngine.JLA.setWispAlive(threadWrapper, false);
                         if (isThreadAsWisp) {
                             ThreadAsWisp.exit(threadWrapper);
                         }
                         if (throwable instanceof CoroutineExitException) {
                             throw (CoroutineExitException) throwable;
                         }
+                        runnable = null;
```

Comment:
pls. add the comments to describe the dependency
```
runnable = null;
```
required by ThreadAsWisp.exit.

Reply:
1. OK
2. Yes it is safer